cmds/storaged/main.go

```diff
@@ -65,6 +66,12 @@ func main() {
 		log.Info().Msg("shutting down")
 	})
 
+	go func() {
+		if err := http.ListenAndServe(":8080", http.DefaultServeMux); err != nil {
```

Comment:
i don't like this port is hard-coded specially a very "common" port like 8080

Reply:
changed it to 8081